src/corona_hakab_model/policies_manager.py

```diff
@@ -78,15 +78,22 @@ def __init__(
             connection_change_factor: float,
             circle_conditions: Iterable[Callable[[Any], bool]],
             agent_conditions: Iterable[Callable[[Any], bool]] = None,
+            policy_props_update: Dict = None,
```

Comment:
```suggestion
            policy_props_update: Dict = dict(),
```

Reply:
giving a param a default mutable value is problematic:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
oh i didn't know about that, sounds good